README.md

```diff
@@ -1,113 +1,143 @@
-# <img src="https://cdn.rawgit.com/theupdateframework/artwork/3a649fa6/tuf-logo.svg" height="100" valign="middle" alt="TUF"/> A Framework for Securing Software Update Systems
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://github.com/shubhusion/python-tuf/blob/develop/tuf-horizontal-white.png">
+  <source media="(prefers-color-scheme: light)" srcset="https://github.com/shubhusion/python-tuf/blob/develop/tuf-horizontal-black.png">
+  <img alt="Shows an illustrated sun in light mode and a moon with stars in dark mode." src="https://github.com/shubhusion/python-tuf/blob/develop/tuf-horizontal-black.png">
+</picture>
+
+
+
+# <div align="center">A Framework for Securing Software Update Systems</div>
 
 ![Build](https://github.com/theupdateframework/python-tuf/actions/workflows/ci.yml/badge.svg)
 [![Coveralls](https://coveralls.io/repos/theupdateframework/python-tuf/badge.svg?branch=develop)](https://coveralls.io/r/theupdateframework/python-tuf?branch=develop)
 [![Docs](https://readthedocs.org/projects/theupdateframework/badge/)](https://theupdateframework.readthedocs.io/)
 [![CII](https://bestpractices.coreinfrastructure.org/projects/1351/badge)](https://bestpractices.coreinfrastructure.org/projects/1351)
 [![PyPI](https://img.shields.io/pypi/v/tuf)](https://pypi.org/project/tuf/)
 [![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/theupdateframework/python-tuf/badge)](https://api.securityscorecards.dev/projects/github.com/theupdateframework/python-tuf)
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+[![License: Apache 2.0](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+---
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [About The Update Framework](#about-the-update-framework)
+- [Key Features](#key-features)
+- [Documentation](#documentation)
+- [Installation](#installation)
+- [Contact](#contact)
+- [Security Issues and Bugs](#security-issues-and-bugs)
+- [License](#license)
+- [Acknowledgements](#acknowledgements)
 
-----------------------------
-[The Update Framework (TUF)](https://theupdateframework.io/) is a framework for
-secure content delivery and updates. It protects against various types of
-supply chain attacks and provides resilience to compromise. This repository is a
-**reference implementation** written in Python. It is intended to conform to
-version 1.0 of the [TUF
-specification](https://theupdateframework.github.io/specification/latest/).
+---
```

Comment:
Now that you are using headers (good idea), I think these horizontal lines are just clutter: I think most of them can just be removed